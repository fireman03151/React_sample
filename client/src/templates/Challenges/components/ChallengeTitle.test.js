/* global expect */

import React from 'react';
import renderer from 'react-test-renderer';

import ChallengeTitle from './Challenge-Title';

const baseProps = {
  children: 'title text',
  isCompleted: true
};

describe('<ChallengeTitle/>', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ChallengeTitle {...baseProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
