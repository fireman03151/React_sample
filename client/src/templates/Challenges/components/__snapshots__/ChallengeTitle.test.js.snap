// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChallengeTitle/> renders correctly 1`] = `
<h2
  className="text-center challenge-title"
>
  title text
  <span
    className="sr-only"
  >
    icons.passed
  </span>
  <svg
    height="50"
    style={
      Object {
        "height": "15px",
        "marginLeft": "5px",
        "width": "15px",
      }
    }
    viewBox="0 0 200 200"
    width="50"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <title>
        icons.passed
      </title>
      <circle
        cx="100"
        cy="99"
        fill="var(--primary-color)"
        r="95"
        stroke="var(--primary-color)"
        strokeDasharray="null"
        strokeLinecap="null"
        strokeLinejoin="null"
      />
      <rect
        fill="var(--primary-background)"
        height="30"
        stroke="var(--primary-background)"
        strokeDasharray="null"
        strokeLinecap="null"
        strokeLinejoin="null"
        transform="rotate(-45, 120, 106.321)"
        width="128.85878"
        x="55.57059"
        y="91.32089"
      />
      <rect
        fill="var(--primary-background)"
        height="30"
        stroke="var(--primary-background)"
        strokeDasharray="null"
        strokeLinecap="null"
        strokeLinejoin="null"
        transform="rotate(45, 66.75, 123.75)"
        width="80.66548"
        x="26.41726"
        y="108.75"
      />
    </g>
  </svg>
</h2>
`;
